src/Microsoft.AspNetCore.Sockets.Client.Http/ServerSentEventsTransport.cs

```diff
@@ -140,6 +137,16 @@ public Task StartAsync(Uri url, Channel<byte[], SendMessage> application, Transf
                 _transportCts.Cancel();
                 stream.Dispose();
                 _logger.ReceiveStopped(_connectionId);
+                pipe.Writer.Complete();
+
+                try
+                {
+                    await writeTask;
+                }
+                catch (Exception ex)
```

Comment:
I looked at code [here](https://github.com/dotnet/corefxlab/blob/master/src/System.IO.Pipelines.Extensions/StreamExtensions.cs#L71-L83) and it seems that exception handling is actually not needed. Also, the writer will be completed so it seems that the only thing to do is to await the task...